nautobot/wireless/choices.py

```diff
@@ -216,130 +216,130 @@ class RadioProfileRegulatoryDomainChoices(ChoiceSet):
-        (MA, "Morocco"),
-        (MC, "Monaco"),
-        (MD, "Moldova"),
-        (MK, "Macedonia"),
-        (MO, "Macau"),
-        (MQ, "Martinique"),
-        (MT, "Malta"),
-        (MU, "Mauritius"),
-        (MX, "Mexico"),
-        (MY, "Malaysia"),
-        (NA, "Namibia"),
-        (NG, "Nigeria"),
-        (NI, "Nicaragua"),
-        (NL, "Netherlands"),
-        (NO, "Norway"),
-        (NZ, "New Zealand"),
-        (OM, "Oman"),
-        (PA, "Panama"),
-        (PE, "Peru"),
-        (PL, "Poland"),
-        (PH, "Philippines"),
-        (PK, "Pakistan"),
-        (PR, "Puerto Rico"),
-        (PT, "Portugal"),
-        (PY, "Paraguay"),
-        (QA, "Qatar"),
-        (RO, "Romania"),
-        (RU, "Russia"),
-        (SA, "Saudi Arabia"),
-        (SE, "Sweden"),
-        (SG, "Singapore"),
-        (SI, "Slovenia"),
-        (SK, "Slovakia"),
-        (SM, "San Marino"),
-        (SV, "El Salvador"),
-        (SY, "Syria"),
-        (TH, "Thailand"),
-        (TN, "Tunisia"),
-        (TR, "Turkey"),
-        (TT, "Trinidad and Tobago"),
-        (TW, "Taiwan"),
-        (UA, "Ukraine"),
-        (US, "United States"),
-        (UY, "Uruguay"),
-        (UZ, "Uzbekistan"),
-        (VA, "Vatican City"),
-        (VE, "Venezuela"),
-        (VI, "Virgin Islands"),
-        (VN, "Vietnam"),
-        (YE, "Yemen"),
-        (ZA, "South Africa"),
-        (ZW, "Zimbabwe"),
+        (AD, "Andorra (AD)"),
+        (AE, "United Arab Emirates (AE)"),
+        (AL, "Albania (AL)"),
+        (AM, "Armenia (AM)"),
+        (AU, "Australia (AU)"),
+        (AR, "Argentina (AR)"),
+        (AT, "Austria (AT)"),
+        (AZ, "Azerbaijan (AZ)"),
+        (BA, "Bosnia and Herzegovina (BA)"),
+        (BE, "Belgium (BE)"),
+        (BG, "Bulgaria (BG)"),
+        (BH, "Bahrain (BH)"),
+        (BN, "Brunei Darussalam (BN)"),
+        (BO, "Bolivia (BO)"),
+        (BR, "Brazil (BR)"),
+        (BS, "Bahamas (BS)"),
+        (BY, "Belarus (BY)"),
+        (BZ, "Belize (BZ)"),
+        (CA, "Canada (CA)"),
+        (CH, "Switzerland (CH)"),
+        (CI, "Cote d'Ivoire (CI)"),
+        (CL, "Chile (CL)"),
+        (CN, "China (CN)"),
+        (CO, "Colombia (CO)"),
+        (CR, "Costa Rica (CR)"),
+        (RS, "Serbia (RS)"),
+        (CY, "Cyprus (CY)"),
+        (CZ, "Czech Republic (CZ)"),
+        (DE, "Germany (DE)"),
+        (DK, "Denmark (DK)"),
+        (DO, "Dominican Republic (DO)"),
+        (DZ, "Algeria (DZ)"),
+        (EC, "Ecuador (EC)"),
+        (EE, "Estonia (EE)"),
+        (EG, "Egypt (EG)"),
+        (ES, "Spain (ES)"),
+        (FO, "Faroe Islands (FO)"),
+        (FI, "Finland (FI)"),
+        (FR, "France (FR)"),
+        (GB, "United Kingdom (GB)"),
+        (GE, "Georgia (GE)"),
+        (GI, "Gibraltar (GI)"),
+        (GL, "Greenland (GL)"),
+        (GP, "Guadeloupe (GP)"),
+        (GR, "Greece (GR)"),
+        (GT, "Guatemala (GT)"),
+        (GY, "Guyana (GY)"),
+        (HN, "Hondura (HN)"),
```

Comment:
Not `Honduras`?

Reply:
's' got accidentally dropped, adding back in.